src/Artsy/Relay/SystemQueryRenderer.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import { QueryRenderer, QueryRendererProps } from "react-relay"
+import { OperationBase, OperationDefaults } from "relay-runtime"
+
+export class SystemQueryRenderer<
+  T extends OperationBase = OperationDefaults
```

Comment:
@alloy - I grabbed this from the Relay types, but I'm not sure how this will impact our tslint autofixer or how to adapt for a wrapper component.